package.json

```diff
@@ -4,7 +4,7 @@
   "description": "Ready-to-go Preact starter project powered by webpack.",
   "scripts": {
     "dev": "cross-env NODE_ENV=development webpack-dev-server --inline --hot --progress",
-    "start": "serve build -s -c 1 && node ./tools/start.cli.js",
+    "start": "serve . -s -c 1 && node ./tools/start.cli.js",
```

Comment:
`serve build` doesn't work because there is no html file inside the build folder.
`serve .` works just fine. I tested both npm start and npm run dev.